easybuild/easyconfigs/p/Proteinortho/Proteinortho-5.16b-foss-2018a-Python-3.6.4-Perl-5.26.1.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'Proteinortho'
+version = '5.16b'
+versionsuffix = '-Python-%(pyver)s-Perl-%(perlver)s'
+
+homepage = 'http://www.bioinf.uni-leipzig.de/Software/proteinortho/'
+description = "Proteinortho is a tool to detect orthologous genes within different species."
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['http://www.bioinf.uni-leipzig.de/Software/proteinortho/']
+sources = ['proteinortho_v%(version)s_src.tar.gz']
+checksums = ['d220d4af5ffae8190eec462d8b6c941022ce2927391493b7741c6b7db96a80f2']
+
+dependencies = [
+    ('Perl', '5.26.1'),
+    ('Python', '3.6.4'),
+    ('BLAST+', '2.7.1'),
+]
+
+files_to_copy = [
+    (['proteinortho%(version_major)s_clustering', 'proteinortho%(version_major)s_tree', 'chk_test.pl',
+      'po2tree.pl', 'proteinortho%(version_major)s_clean_edges2.pl', 'proteinortho%(version_major)s.pl',
+      'proteinortho%(version_major)s_singletons.pl', 'ffadj_mcs.py'], 'bin'),
+    'tools', 'manual.html',
+]
+
+sanity_check_paths = {
+    'files': ["bin/proteinortho%(version_major)s_clustering", "bin/proteinortho%(version_major)s_tree", 
```

Comment:
@ebirn Our style bot is tripping over the trailing whitespace here (extra space)...